api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3794,12 +3807,117 @@ public void saveOrderGroup_shouldFailValidationIfAnyOrdersFailValidation() {
 		Exception expectedGroupValidationError = null;
 		try {
 			Context.getOrderService().saveOrderGroup(orderGroup);
-		}
-		catch (Exception e) {
+		} catch (Exception e) {
 			expectedGroupValidationError = e;
 		}
 
 		assertNotNull(expectedGroupValidationError, "Validation should cause order group to fail to save");
 		assertEquals(expectedValidationError.getMessage(), expectedGroupValidationError.getMessage());
 	}
+	
+	@Test
+	public void getOrderGroupAttributeTypes_shouldReturnAllOrderGroupAttributeTypes() {
+		List<OrderGroupAttributeType> orderGroupAttributeTypes = orderService.getAllOrderGroupAttributeTypes();
+		assertEquals(4, orderGroupAttributeTypes.size());
+	}
+	
+	@Test
+	public void getOrderGroupAttributeType_shouldReturnNullIfNoOrderGroupAttributeTypeHasTheGivenId() {
+		assertNull(Context.getOrderService().getOrderGroupAttributeType(10));
+	}
+	
+	@Test
+	public void getOrderGroupAttributeType_shouldReturnOrderGroupAttributeType() {
+		OrderGroupAttributeType orderGroupAttributeType = orderService.getOrderGroupAttributeType(2);
+		assertThat(orderGroupAttributeType.getId(), is(2));
+	}
+	
+	@Test
+	public void getOrderGroupAttributeTypeByUuid_shouldReturnOrderGroupAttributeTypeByUuid() {
+		OrderGroupAttributeType orderGroupAttributeType = orderService
+		        .getOrderGroupAttributeTypeByUuid("9cf1bce0-d18e-11ea-87d0-0242ac130003");
+		assertEquals("Bacteriology", orderGroupAttributeType.getName());
+	}
+	
+	@Test
+	public void saveOrderGroupAttributeType_shouldSaveOrderGroupAttributeTypeGivenOrderGroupAttributeType()
+	        throws ParseException {
+		int initialGroupOrderAttributeTypeCount = Context.getOrderService().getAllOrderGroupAttributeTypes().size();
```

Comment:
Why use Context.getOrderService() in many methods when you already have a class level orderService property?

Reply:
Cleaned out !